src/applications/edu-benefits/1990s/content/directDeposit.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+export const directDepositDescription = () => {
+  return (
+    /* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */
+    <div tabIndex="0">
+      <p>
+        We make payments only through direct deposit, also called electronic
+        funds transfer (EFT). If you’re approved for the Rogers STEM
+        Scholarship, we’ll need to know where to deposit the funds.
```

Comment:
We're not applying for the STEM scholarship

Reply:
Fixed